test/e2e/kubectl/kubectl.go

```diff
@@ -724,6 +724,21 @@ metadata:
 		})
 	})
 
+	framework.KubeDescribe("Kubectl get componentstatuses", func() {
+		It("should get componentstatuses", func() {
+			By("getting list of componentstatuses")
+			output := framework.RunKubectlOrDie("get", "componentstatuses", "-o", "jsonpath={.items[*].metadata.name}")
+			components := strings.Split(output, " ")
+			By("getting details of componentstatuses")
+			for _, component := range components {
+				result := framework.RunKubectlOrDie("get", "componentstatuses", component, "-o", "json")
+				if !strings.Contains(result, component) {
```

Comment:
I don't think this comparison is needed, since you `RunKubectlOrDie` will fail if the component doesn't exist.
Basically, you just need to run the`get componentstatus <component>` to see if this works, the output is irrelevant.

Reply:
Thanks futher comments. I saw the command failed with "Component not found" as you said.
I changed to only execute RunKubectlOrDie. Also, I removed "-o json", because I eliminated comparison of the outputs. 
Could you review again?